WORKSPACE

```diff
@@ -8,3 +8,7 @@ fhirproto_workspace()
 
 load("@fhir_bazel_pip_dependencies//:requirements.bzl", "pip_install")
 pip_install()
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_rules_dependencies", "go_register_toolchains")
```

Comment:
I'd prefer it (for other languages' sake, that it would be possible to have this golang dependency be opt-in. Is there a mechanism whereby this could be achieved? Would it be a good idea in general to split the language libraries out, giving you the raw proto, then child java, cc and golang libraries on top of that?

Reply:
I would definitely be open to that, it makes a lot of sense. I am a very new to Bazel, so any thoughts/examples on how to do this would be appreciated.

Reply:
@kyle-winkelman I believe an approach like google's googleapis repository might be a good bet https://github.com/googleapis/googleapis/blob/master/repository_rules.bzl#L56.

Here they have a repository rule `switched_rules_by_language` which you call in a WORKSPACE with the particular language implementations (de)selected in order to build the proto specs you wish to build.